starlette/datastructures.py

```diff
@@ -286,7 +290,7 @@ def get(self, key: typing.Any, default: typing.Any = None) -> typing.Any:
             return self._dict[key]
         return default
 
-    def __getitem__(self, key: typing.Any) -> str:
+    def __getitem__(self, key: typing.Any) -> _VT:
```

Comment:
Perhaps we could just `-> Any` this?

Not a big fan of the `TypeVar` cases here - gets into the zone of Python typing that I'd prefer we just avoid where possible.

Reply:
I'm open to it. We can always add stronger types later on.

Reply:
done in bd33808